### ISSUE TITLE
[FEATURE REQUEST] New game sorting options on the account

### ISSUE THREAD

**Issue by lucasbooz** [IsThereAnyDeal/AugmentedSteam#1541]
### Current Behavior (if any)
Currently it is not possible to access the Steam library through the browser, but it is possible to check all the games that I have on the account. However, it is not possible to apply the same sorters to games that the desktop version of Steam has, such as "sort by: Steam reviews, Metacritic score and release date".
### Desired Behavior
![image](https://user-images.githubusercontent.com/67797271/205913898-71a0437c-dbfc-4773-8ba4-3121307d5f37.png)
It would be interesting to add at least the following orderers below:
![image](https://user-images.githubusercontent.com/67797271/205915898-b40d9203-cb46-4fd0-8d5e-9e8ad74c671b.png)
### Motivation / Use Case for Changing the Behavior
The web version of Steam does not have all the filters noted above, and I believe it is essential to add these new options.

**Comment by candela97** [comment]
Yeah we have a few suggestions like this, the problem is this data is not directly accessible from the page, which means we'll need to fetch it from elsewhere, which will amount to a lot of requests, making it unusable. (I think this page is already too slow to be usabe for people with over a thousand games 😅)
